bootstrap/helpers/services.php

```
@@ -339,3 +340,54 @@ function parseServiceEnvironmentVariable(string $key): array
+                    $application->save();
+                }
+            }
+        }
+    } catch (\Throwable $e) {
+        // Log error but don't throw - prerequisites are nice-to-have, not critical
+        Log::error('Failed to apply service application prerequisites', [
+            'service_id' => $service->id,
+            'service_name' => $service->name,
+            'error' => $e->getMessage(),
+            'trace' => $e->getTraceAsString(),
+        ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'applyServiceApplicationPrerequisites'. (undefined)

(StaticAccess)